Mathlib/CategoryTheory/Bicategory/LocallyDiscrete.lean

```diff
@@ -106,6 +106,9 @@ instance locallyDiscreteBicategory.strict : Strict (LocallyDiscrete C) where
   comp_id f := Discrete.ext (Category.comp_id _)
   assoc f g h := Discrete.ext (Category.assoc _ _ _)
 
+attribute [local simp]
+  Strict.leftUnitor_eqToIso Strict.rightUnitor_eqToIso Strict.associator_eqToIso
```

Comment:
See my comments in [zulip](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/.60comp_id.60.20in.20.60Cat.60).